src/config/nodemailer.ts

```diff
@@ -0,0 +1,18 @@
+import { createTransport } from "nodemailer";
+
+const email: string | undefined = process.env.EMAIL;
+const pass: string | undefined = process.env.EMAIL_PASS;
```

Comment:
these should be added to `src/backend/lib/types/env.ts`

also would be good to document them in `.env.example`